### PR TITLE
Added sorting of SGs and rules

### LINKED ISSUE #469: Repeatable import/output compatible with diff
With the following steps:

 - import resources (ex. `generated/aws/sg/security_group.tf`)
 - commit files to git
 - import resources again
 - git diff generated/aws/sg/security_group.tf

I get a long list of changes, even if I only changed a single ingress rule, because the output is not sorted the same way. Even running an import again with no changes may result in changed output.

This issue is a request to change the method by which the import files are written, such that the files are as identical as possible when the import is repeated.

### PR DESCRIPTION
**Pull request by meshuga**
It's AWS issue to return data in unordered manner but fortunately, that's fine to pass data in random order to AWS.

closes #469

### CODE REVIEW

File: providers/aws/sg.go [hunk 1]
```diff
@@ -238,9 +238,14 @@ func (g *SecurityGenerator) InitResources() error {
 	}
 	svc := ec2.New(config)
 	p := ec2.NewDescribeSecurityGroupsPaginator(svc.DescribeSecurityGroupsRequest(&ec2.DescribeSecurityGroupsInput{}))
+	var resourcesToFilter []ec2.SecurityGroup
 	for p.Next(context.Background()) {
-		g.Resources = append(g.Resources, g.createResources(p.CurrentPage().SecurityGroups)...)
+		resourcesToFilter = append(resourcesToFilter, p.CurrentPage().SecurityGroups...)
 	}
+	sort.Slice(resourcesToFilter[:], func(i, j int) bool {
```
Comment (sergeylanzman) [inline providers/aws/sg.go:245]: Why you need `[:]`?

File: providers/aws/sg.go [hunk 2]
```diff
@@ -238,9 +238,14 @@ func (g *SecurityGenerator) InitResources() error {
...
+		return *resourcesToFilter[i].GroupId < *resourcesToFilter[j].GroupId
```
Comment (sergeylanzman) [inline providers/aws/sg.go:246]: I think it's need `resourcesToFilter[i].GroupId < resourcesToFilter[j].GroupId`

File: providers/aws/sg.go (hunk 1 above)
Comment (meshuga) [reply providers/aws/sg.go:245]: Good question, works well even without defining a slice :)

File: providers/aws/sg.go (hunk 2 above)
Comment (meshuga) [reply providers/aws/sg.go:246]: I want to compare strings, not pointers

**Pull request merged**